.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,26 @@
-<!--- Provide a short summary of your changes in the Title above -->
+<!--- Provide a short summary of your changes in the Title above. -->
 
 ## Description
-<!--- Describe your changes in detail, what problems does it solve? -->
+<!--- Describe your changes in detail; what problems does it solve? -->
 
 ## Related Issue
-<!--- If you are suggesting a new feature or change, please create an issue first -->
-<!--- Please link to the issue, discourse, or stackoverflow here: -->
+<!--- If you are suggesting a new feature or change, please create an issue first. -->
+<!--- Please link to the issue, discourse, or stackoverflow here. -->
+
+## Definition of Done
+<!--- Explain the goal to reviewers so they know what the target is supposed to be. -->
+
+## How to Build and Test the Change
```

Comment:
This really feels like we're adding a lot of burden on OSS contributors by requiring potentially more work filling this out than making the change. I get the intention here is for internal users, but this gets applied to several hundred repos.

Reply:
Fair point; would you rather omit it or word it so that it could be optional?